test/jquery.dbEditorCombo.test.js

```diff
@@ -0,0 +1,111 @@
+beforeEach(() => {
+    require('./mocks/offsetParent.js');
```

Comment:
Mocks have a specific meaning with Jest 
https://jestjs.io/docs/en/mock-functions

Can you justify using this term ?
Is your offsetParent function a polyfill ?

Reply:
Updated and re-tested with folder renamed to "polyfills"